python-lib/plugin_io_utils.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
```

Comment:
So this module should be split in two:
- plugin_io_utils.py which does *not* use the Dataiku API
- dku_io_utils.py which does use the Dataiku API

You can check the structure of https://github.com/dataiku/dss-plugin-nlp-preparation/tree/master/python-lib for reference

The reason is that you should cleanly separate DSS specific stuff from "Python stuff"

DSS stuff can only be executed and tested in the context of DSS while Python stuff can be unit-tested outside.

Reply:
had copied the list from another plugin, there's only spacy_tokenizer calling this file so removed the DSS linked stuff and no dku_io_utils.py needed